cypress-tests/cypress.config.ts

```diff
@@ -3,7 +3,11 @@ import { TIMEOUTS } from "./config";
 
 export default defineConfig({
   e2e: {
-    baseUrl: process.env.BASE_URL || "https://dev.renku.ch"
+    baseUrl: process.env.BASE_URL || "https://dev.renku.ch",
+    specPattern: [
+      "cypress/e2e/verifyInfrastructure.cy.ts",
```

Comment:
I think a comment here would be helpful to ensure that no one accidentally reorders this.

```suggestion
      // Verify infrastructure before running any other tests
      "cypress/e2e/verifyInfrastructure.cy.ts",
```

Reply:
Good idea! :+1: